src/sanitizer.js

```diff
@@ -156,7 +156,7 @@ const BLACKLISTED_ATTR_VALUES = [
 /** @const {!Object<string, !Object<string, !RegExp>>} */
 const BLACKLISTED_TAG_SPECIFIC_ATTR_VALUES = dict({
   'input': {
-    'type': /(?:image|file|password|button)/i,
+    'type': /(?:image|file|button)/i,
```

Comment:
`amp-form.md` files needs updating as well. It mentions password is not allowed.